packages/cdktf-cli/templates/typescript/cdktf.json

```diff
@@ -1,9 +1,5 @@
 {
-  "language": "typescript",
   "app": "npm run --silent compile && node main.js",
-  "terraformProviders": [
```

Comment:
I think we should make this template consistent with the others and have empty arrays. Maybe also keep the language to clarify what needs to change if switching languages.

Reply:
Was lost while copying `typescript-minimal` template. added it again.